website/docs/docs/getting-started/2-your-first-test.md

```diff
@@ -0,0 +1,41 @@
+---
+id: your-first-test
+title: Writing your first test
+---
+
+To demonstrate how you can write tests, let's install the TodoMVC app from BigTest:
+```
+$ yarn add -D @bigtest/todomvc
```

Comment:
@cowboyd is there a way we can extract `todomvc` from the monorepo?  I think it'd be better if we renamed it `accessible-todomvc` for example, instead of linking it to bigtest directly. In my opinion, the current setup smells fishy because you're using a bigtest app to test bigtest. Makes me wonder if it'll run on any other app or not.

Plus, we hint a lot of stuff on the `accesible-todomvc` name: that we care a11y, that your app must follow good practices to use bigtest